src/main/scala/org/broadinstitute/dsde/workbench/gpalloc/db/ActiveOperationComponent.scala

```diff
@@ -0,0 +1,58 @@
+package org.broadinstitute.dsde.workbench.gpalloc.db
+
+import org.broadinstitute.dsde.workbench.gpalloc.model.BillingProjectStatus
+import org.broadinstitute.dsde.workbench.gpalloc.model.BillingProjectStatus.BillingProjectStatus
+
+case class ActiveOperationRecord(billingProjectName: String,
```

Comment:
This may deserve its own file for clarity.

Reply:
records live with their corresponding components